src/loader/bgw_launcher.c

```diff
@@ -20,6 +20,9 @@
 #include <utils/snapmgr.h>
 #include <access/xact.h>
 
+/* and checking db list for whether we're in a template*/
```

Comment:
This comment appears to have lost the beginning?

Reply:
it was meant to continue the stuff from getting a db list, cause we need a few of those for checking whether we're in a template as well...but that might not be so clear.